packages/tsconfig-reference/copy/pt/categories/Additional_Checks_6176.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
## TSConfig Support

These are the names of the category of compiler options, as well as an overview of the grouping.